docs/guide.installation.rst

```diff
@@ -126,17 +180,52 @@ Press ``ctrl+x`` to add Hardware, and fill in the relevant parameters (most are
     :alt: Configure Hardware
     :width: 100%
 
+After completing this step, the file `prefs.json` will be created if necessary and populated with the information you just provided. If it already exists, it will modified with the new information while preserving the previous preferences. 
 
-Supported Systems
-===================
+You can also manually edit the prefs.json file if you prefer. `A template version for the Pilot is available <https://groups.google.com/g/autopilot-users/c/_MqzLDDq3CE>`_ that defines the ports, LEDs, and solenoids that are necessary for the "free water" task, which may be a useful way to get started.
 
-Autopilot is **linux/mac** only, and supports **Python 3.7**. Some parts might accidentally work in Windows but we make no guarantees.
+Networking
+==========
+The Pilot and Terminal devices must be on the same network and capable of reaching one another. You must first figure out the IP address of each device with this command::
 
-In particular, the Terminal was designed for Ubuntu, and the Pilot was designed for the Raspberry Pi OS.
+    ipconfig
 
-We have tried to take care to make certain platform-specific dependencies not break the entire package,
-so if you have some difficulty installing autopilot on a non-raspberry-pi linux machine please submit an issue!
+Let's say your Terminal is at 192.168.1.42 and your Pilot is at 192.168.1.200. Replace these values with whatever you actually find using `ipconfig`.
+
+Then, you can test that each device can see the other with ping. On the Terminal, run::
+
+    ping 192.168.1.200
+
+And on the Pilot, run::
+
+    ping 192.168.1.42
+
+If that doesn't work, something's up with your router or building network.
+
+You also need to make sure that a port is open on the Terminal device to receive messages from the Pilot. By default, this is port 5560. If you don't have a firewall, your ports are already open. But your version of Ubuntu may have a firewall ("ufw") running. If so, you can open this port on the Terminal by running the following::
+
+    sudo ufw allow from 192.168.1.200 to any port 5560
+
+Make sure that the Pilot knows where the Terminal is by editing the line TERMINALIP in prefs.json on the Pilot to 192.168.1.42 (or whatever the IP address of your Terminal is).
+
+Similarly, tell the Terminal to expect information on port 5560. Do this by editing prefs.json on the Terminal and setting MSGPORT to 5560. Note that MSGPORT/PUSHPORT should be swapped between Terminal and Pilot devices.
+
+
+Testing the Installation
+========================
+
+To test whether this all worked, try to start Autopilot. (Don't forget to activate your virtual environment first!) 
+
+Run this on the Pilot::
+
+    python3 -m autopilot.core.pilot -f ~/autopilot/prefs.json
+
+Alternatively, there is a launch script for the Pilot that includes some other magic, like activating the virtual environment and killing some troublesome daemons.::
```

Comment:
;) this is the expected entrypoint now, and i added a script to make an alias `autopilot` that points to it. the plan is to unify agent structure and launch from a single place, but keeping the launch script for now